Makefile

```diff
@@ -36,6 +36,16 @@ include .make/sdk/main.make
 messages:
 	@$(GO) run ./cmd/internal/generate_i18n.go
 
+docs-gen:
+	hugo -s ./doc --baseUrl https://thethingsnetwork.github.io/lorawan-stack/$(GIT_TAG)/ -d public/$(GIT_TAG)
+
+docs-server:
+	hugo server -s ./doc
+
+docs-deps:
+	git submodule init
```

Comment:
remove in favor of https://github.com/TheThingsNetwork/lorawan-stack/pull/395/files#r275362929